index.html

```diff
@@ -22,7 +22,12 @@
     <div class="header">
       <header class="container">
         <a class="navbar" href="">My Logo</a>
-        <nav class="button"><i class="fa-solid fa-bars"></i></nav>
+        <nav class="button button1"><i class="fa-solid fa-bars"></i></nav>
+        <ul class="orders">
+          <li>Portoflio</li> 
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
       </header>
     </div>
 
```

Comment:
- [ ] Great job creating your desktop version! However, there are a few things that need to be fixed:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-01 a la(s) 9 16 51 p  m](https://user-images.githubusercontent.com/83717285/171535606-58263a00-e9ff-4cbd-a2f2-7c039633b668.png)|![Captura de pantalla 2022-06-01 a la(s) 9 16 31 p  m](https://user-images.githubusercontent.com/83717285/171535725-4a37c07b-272d-4d20-9b2a-00af14911674.png)|

- [ ] I would suggest the font from the Figma template for the navigation links, as this is a requirement.
- [ ] It would be better to increase the width of the paragraph, in order to make it similar to the Figma template.
- [ ] Background is getting stretched, it would be great to display it correctly as in the template.
- [ ] Let's move the content more to the left as shown in the original design.

Reply:
every thing is done